acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/web3j/privacy/PrivacyReceiptAcceptanceTest.java

```diff
@@ -81,6 +84,37 @@ public void createPrivateTransactionReceiptFailedTransaction() {
     alice.getBesu().verify(priv.getFailedTransactionReceipt(transactionHash));
   }
 
+  @Test
+  public void createPrivateTransactionReceiptInvalidTransaction() {
+    final CreatePrivacyGroupTransaction onlyAlice =
+        privacyTransactions.createPrivacyGroup("Only Alice", "", alice);
+
+    final String privacyGroupId = alice.execute(onlyAlice);
+
+    final PrivateTransaction invalidPayloadTransaction =
+        createSignedTransaction(alice, privacyGroupId, empty());
+    final BytesValueRLPOutput rlpOutput = getRLPOutput(invalidPayloadTransaction);
+
+    // Stop mining, to allow adding duplicate nonce block
+    alice.getBesu().execute(minerTransactions.minerStop());
+
+    final Hash transactionHash1 =
+        alice.execute(privacyTransactions.sendRawTransaction(rlpOutput.encoded().toHexString()));
+    final Hash transactionHash2 =
+        alice.execute(privacyTransactions.sendRawTransaction(rlpOutput.encoded().toHexString()));
+
+    // Start mining again
+    alice.getBesu().execute(minerTransactions.minerStart());
+
+    // Successful PMTs
+    alice.getBesu().verify(eth.expectSuccessfulTransactionReceipt(transactionHash1.toString()));
+    alice.getBesu().verify(eth.expectSuccessfulTransactionReceipt(transactionHash2.toString()));
+    // Successful first private transaction
+    alice.getBesu().verify(priv.getSuccessfulTransactionReceipt(transactionHash1));
+    // Invalid second private transaction
+    alice.getBesu().verify(priv.getInvalidTransactionReceipt(transactionHash2));
```

Comment:
I can see a test for a successful private tx and for an invalid private tx. What about a failed private tx?

Reply:
That's in the previous test.